recipe/meta.yaml

```
@@ -1,25 +1,25 @@
 {% set name = "openff-units" %}
-{% set version = "0.2.0" %}
+{% set version = "0.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/openforcefield/openff-units/archive/{{ version }}.tar.gz
-  sha256: ed4fbb20fbf9889123785a3f753343526e815001533780d1267110c3752f9daa
+  sha256: 35578e85a82ac54195462b8dde95e0ab51330978e19ee9789bd0a25aa05af4dc
 
 build:
   noarch: python
-  number: 2
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
     - pip
   run:
-    - python >=3.8
+    - python >=3.9
     - numpy
     - pint =0.21
```

```suggestion
    - pint =0.21|0.22
```
I'm not sure the toolkit is ready for this - @j-wags?

---

It's not - I'd appreciate if this pin could stick around until I cut OFFTK 0.13.2 - I could get that out in the next few days if needed!

---

Since that's a toolkit constraint, could we just constrain it there?

https://github.com/conda-forge/openff-toolkit-feedstock/pull/53

---

This would fix the issue at hand and not depend on a release or other moving pieces